LuaUI/widgets/chobby/components/chat_windows.lua

```diff
@@ -342,3 +364,67 @@ function ChatWindows:GetPrivateChatConsole(userName)
 
     return privateChatConsole
 end
+
+function ChatWindows:JoinChannel()
+
+
+	self.channelName = EditBox:New {
+		bottom = 50,
+		height = 25,
+		right = 50,
+		text = "",
+	}
+
+    self.joinWindow = Window:New {
+        right = 300,
+        width = 150,
+        bottom = 200,
+        height = 100,
+        parent = screen0,
+        caption = i18n("channel"),
+        resizable = false,
+        draggable = false,
+        padding = {5, 0, 5, 0},
+        children = {
+			self.channelName,
+            Button:New {
+                caption = i18n("join"),
+                width = 60,
+                bottom = 4,
+                right = 2,
```

Comment:
the positive action (OK, Yes, join, etc.) should always be on the left side, while the negative (Cancel, No, etc.) should be on the right